frontend/src/entry/model/Files.tsx

```diff
@@ -0,0 +1,65 @@
+import { Card, Container, Stack } from '@mui/material'
+import { useGetModelFiles } from 'actions/model'
+import { useMemo } from 'react'
+import EmptyBlob from 'src/common/EmptyBlob'
+import Loading from 'src/common/Loading'
+import FileDownload from 'src/entry/model/releases/FileDownload'
+import MessageAlert from 'src/MessageAlert'
+import { EntryInterface, FileInterface } from 'types/types'
+
+type FilesProps = {
+  model: EntryInterface
+}
+
+export default function Files({ model }: FilesProps) {
+  const { entryFiles, isEntryFilesLoading, isEntryFilesError } = useGetModelFiles(model.id)
+
+  const sortEntryFiles: Array<FileInterface> = useMemo(
```

Comment:
Minor, but `sortEntryFiles` sounds like you're calling a function to sort the files. It's a variable that stores the sorted files, so consider renaming to `sortedEntryFiles`